fda/datasets/_real_datasets.py

```diff
@@ -387,3 +386,115 @@ def fetch_medflies(return_X_y: bool=False):
+        "CanadianWeather", "fda",
+        version="2.4.7")
+
+    data = raw_dataset["CanadianWeather"]
+
+    weather_daily = np.asarray(data["dailyAv"])
+
+    temp_prec_daily = np.transpose(weather_daily[:, :, 0:2], axes=(1, 0, 2))
+
+    curves = FDataGrid(data_matrix=temp_prec_daily,
+                       sample_points=range(1, 366),
+                       dataset_label="Canadian Temperatures",
+                       axes_labels=["day", "temperature (ºC)", "precipitation (mm.)"])
+
+    target = data["region"]
+
+    if return_X_y:
+        return curves, target
+    else:
+        return {"data": curves,
+                "target": target,
+                "target_names": np.unique(target),
+                "target_feature_names": ["region"],
+                "DESCR": DESCR}
+
+
+if hasattr(fetch_weather, "__doc__"):  # docstrings can be stripped off
+    fetch_weather.__doc__ += _weather_descr + _param_descr
+
+_aemet_descr = """
+    Series of daily summaries of 73 spanish weather stations selected for the period 1980-2009.  The
+    dataset contains geographic information of each station and the average for the period 1980-2009
+    of daily temperature, daily precipitation and daily wind speed.
+    Meteorological State Agency of Spain (AEMET), http://www.aemet.es/. Government of Spain.
+
+    Authors:
+        Manuel Febrero Bande, Manuel Oviedo de la Fuente <manuel.oviedo@usc.es>
+
+    Source:
+        The data were obtained from the FTP of AEMET in 2009.
+"""
+
+
+def fetch_aemet(return_X_y: bool = False):
+    """
+    Load the Spanish Weather dataset.
+
+    The data is obtained from the R package 'fda.usc' from CRAN.
+
+    """
+    DESCR = _aemet_descr
+
+    raw_dataset = fetch_cran(
+        "aemet", "fda.usc",
+        version="1.3.0")
+
+    data = raw_dataset["aemet"]
+
+    fd_temp = data["temp"]
+    fd_logprec = data["logprec"]
+    fd_wind = data["wind.speed"]
+
+    regions = np.unique(data['df']['province'])
+    climate_regions = ["OCEANIC", "OCEANIC", "CONTINENTAL", "MEDITERRANEAN", "ARID", "OCEANIC", "CONTINENTAL",
+                       "MEDITERRANEAN", "MEDITERRANEAN", "CONTINENTAL", "MEDITERRANEAN", "OCEANIC", "MEDITERRANEAN",
+                       "CONTINENTAL", "MEDITERRANEAN", "CONTINENTAL", "MEDITERRANEAN", "MEDITERRANEAN", "CONTINENTAL",
+                       "OCEANIC", "CONTINENTAL", "CONTINENTAL", "CANARY", "CONTINENTAL", "CONTINENTAL", "MEDITERRANEAN",
+                       "MEDITERRANEAN", "ARID", "CONTINENTAL", "OCEANIC", "OCEANIC", "CONTINENTAL",
+                       "CANARY", "MEDITERRANEAN", "CONTINENTAL", "MEDITERRANEAN", "MEDITERRANEAN",
+                       "CONTINENTAL", "OCEANIC", "CONTINENTAL", "CONTINENTAL"]
+    climate_region_dict = dict(zip(regions, climate_regions))
+    climate_data = np.copy(data['df']['province'])
+    for k, v in climate_region_dict.items():
+        climate_data[climate_data == k] = v
+    climate_data[data['df']['altitude'] > 1500] = "MOUNTAIN"
```

Comment:
Why the original dataset is incomplete?

Reply:
Incomplete? What I am doing in that line is classify the locations with altitudes above 1500m as Mountain climate.